CONTRIBUTING.md

```diff
@@ -53,7 +53,31 @@ Before contributing code or documentation to this project, make sure you read th
 
 ### Commit message standards
 
-The commit message should contain an overall explanation about the change and the motivation behind it. Please note that mentioning a Jira ticket ID or a GitHub issue, isn't a replacement for that.
+The commit message should contain an overall explanation about the change and the motivation behind it.
+
+In addition to an explanation about the change, please ensure that each commit contains a line similar to one of the following:
+
+|if your commit|use this keyword|value|
+|-|-------|-----|
+|resolves an issue/ticket|resolves, res, resolved| #123(only for GitHub Issues), PROJ-1234 (any other issue tracker)|
+|only references an issue/ticket|ref, refs, references, refers to| #123(only for GitHub Issues), PROJ-1234 (any other issue tracker)|
+
+*Example git commit message*:
+```
+Fixed the pesky bug.
+
+This commit fixed that pesky bug that we all hated.
+
+resolves: PROJ-1234
+Signed-off-by: John Doe <jdoe@example.com>
```

Comment:
Are we mandating Sign offs?

Reply:
No, this was for example purposes.

Reply:
Perhaps we can remove this then?
```suggestion
```

Reply:
If it encourages folks to sign-off, so be it. We _should_ probably start mandating sign-offs.